content/40.inequality.md

```diff
@@ -182,31 +254,32 @@ This approach would carry its own ethical concerns, including the fact that many
 As the pandemic has progressed, it has become clear that ICU beds and ventilators are not the only limited resources that needs to be allocated, and, in fact, the survival rate for patients who receive mechanical ventilation is lower than these discussions would suggest [@doi:10.1136/medethics-2020-106460].
 Allocation of interventions that may reduce suffering, including palliative care, has become critically important [@doi:10.1136/medethics-2020-106460; @doi:10.1080/15265161.2020.1788663].
 The ambiguities surrounding the risks and benefits associated with therapeutics that have been approved under emergency use authorizations also present ethical concerns related to the distribution of resources [@doi:10.1080/15265161.2020.1795538].
-For example, remdesivir, discussed above, is currently available for the treatment of COVID-19 under compassionate use guidelines and through expanded access programs, and in many cases has been donated to hospitals by Gilead [@doi:10.1016/j.mayocp.2020.06.016; @doi:10.1080/15265161.2020.1795529].
+For example, for much of 2020, remdesivir was available for the treatment of COVID-19 under compassionate use guidelines and through expanded access programs, and in many cases has been donated to hospitals by the manufacturer, Gilead [@doi:10.1016/j.mayocp.2020.06.016; @doi:10.1080/15265161.2020.1795529].
+Remdesivir is now the only FDA-approved treatment for COVID-19.<!--To Do: cite therapeutics-->
```

Comment:
```suggestion
Remdesivir is now the only FDA-approved treatment for COVID-19 [@individual-pharmaceuticals].
```
This will work after #891 is merged.